leapp.spec

```diff
@@ -0,0 +1,121 @@
+Name:       leapp
+Version:    0.0.1
+Release:    3
+Summary:    leapp tool rpm
+
+Group:      Unspecified
+License:    GPLv2+
+URL:        https://github.com/leapp-to/prototype
+Source0:    https://github.com/leapp-to/prototype/archive/master.tar.gz
+BuildArch:  noarch
+
+BuildRequires:   python2-devel
+%if 0%{?el7}
+BuildRequires:   python-setuptools
+%else
+BuildRequires:   python2-setuptools
+%endif
+
+BuildRequires:   libvirt-python
+BuildRequires:   libvirt-devel
+BuildRequires:   python-enum34
+BuildRequires:   python2
+BuildRequires:   python2-nmap
+BuildRequires:   python2-paramiko
```

Comment:
Oh right it might not be necessary anymore because I am messing with the setup.py during the build to NOT require something